xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -923,6 +923,21 @@ void CGUIDialogPVRTimerSettings::PrioritiesFiller(
     list.clear();
     pThis->m_timerType->GetPriorityValues(list);
     current = pThis->m_iPriority;
+
+    auto it = list.begin();
```

Comment:
You're right, you'd have to use `std::find_if`.

```
auto it = std::find_if(list.begin(), list.end(), [current](std::pair<foo, bar> &mapEntry) {
    return mapEntry.second == current;
}
```

Up to which way to do it, the current approach is okay too.